Source/Particles/ERFPCEvolve.cpp

```diff
@@ -130,10 +129,16 @@ void ERFPC::AdvectWithFlow ( MultiFab*                           a_umac,
                     iv[1] += domain.smallEnd()[1];
```

Comment:
@AMLattanzi @asalmgren Can lines 124-129 be replaced by `ERFParticlesAssignor(p, plo, dxi, domain)` (defined in ERFPC.H)? Same with lines 226-231 below?